sdcm/scan_operation_thread.py

```diff
@@ -490,12 +490,18 @@ def _validate_fullscan_result(
         output = "\n".join([str(i) for i in result])
         if int(result[0].count) <= 0:
             return f"Fullscan failed - count is not bigger than 0: {output}", Severity.ERROR
-        get_query_trace = cmd_result.get_query_trace().events
-        for trace_event in get_query_trace:
-            if dispatch_forward_statement_regex_pattern.search(str(trace_event)):
-                regex_found = True
-                break
-        self.log.debug("Fullscan aggregation result: %s", output)
+
+        get_query_trace = []
+        try:
+            get_query_trace = cmd_result.get_query_trace().events
+            for trace_event in get_query_trace:
+                if dispatch_forward_statement_regex_pattern.search(str(trace_event)):
+                    regex_found = True
+                    break
+            self.log.debug("Fullscan aggregation result: %s", output)
+        except Exception as exp:  # pylint: disable=broad-except
+            self.log.warning(exp)
```

Comment:
What would we do with all those warnings ? And print of stack traces to stdout ?

Why not remove the code ? or leave it in comment